ir/src/main/scala/org/scalajs/core/ir/Trees.scala

```diff
@@ -869,6 +869,37 @@ object Trees {
      */
     final case class Global(path: List[String]) extends JSNativeLoadSpec
 
+    /** Load from a module import.
+     *
+     *  The `module` is the ES module identifier. The `path` is a series of
+     *  nested property names starting from the module object.
+     *
+     *  The path can be empty, in which case the specification denotes the
+     *  module itself (whose exports are available as fields).
+     *
+     *  Any element in the path is a property selection from there. A module
+     *  import info with one path element is importing that particular value
+     *  from the module.
+     *
+     *  Examples:
+     *  {{{
+     *  // import { Bar as x } from 'foo'
+     *  Import("foo", List("Bar"))
+     *
+     *  // import { Bar as y } from 'foo'
+     *  // y.Baz
+     *  Import("foo", List("Bar", "Baz"))
+     *
+     *  // import * as x from 'foo' (whole module import)
+     *  Import("foo", Nil)
+     *
+     *  // import x from 'foo' (default import)
+     *  Import("foo", List("default"))
```

Comment:
Relevant information: how Node.js plans to deal with ES importing legacy: https://github.com/nodejs/node-eps/blob/master/002-es6-modules.md#54-es-consuming-commonjs

> `module.exports` is a single value. As such it does not have the dictionary like properties of ES module exports. In order to facilitate named imports for ES modules, all properties of `module.exports` will be hoisted to named exports after evaluation of CJS modules with the exception of `default` which will point to `module.exports` directly.

That interpretation *does* hide a property actually called `default`, as the first example in that section shows. This is not the same interpretation as Rollup. Note that this interpretation is virtually impossible to polyfill properly (when importing `default`, we don't know whether it should be the module itself (if it's legacy) or its default export (if it's ES)).

Irrelevant information: the current state of the Browser Loader spec does not seem to mention `default` anywhere: https://whatwg.github.io/loader/
To be fair, though, it *seems* that it does not *allow* to import non-ES modules at all, which bypasses the issue.